website/docs/user_guides/overview_of_chaos_dashboard.md

```diff
@@ -0,0 +1,150 @@
+---
+id: overview_of_chaos_dashboard
+title: Overview of Chaos Dashboard
+---
+
+This document will give you an overview of Chaos Dashboard. We will simplify some of the steps we saw in [Run Chaos Experiment](run_chaos_experiment) and guide you to use the visual interface based on our existing features.
+
+We are going to talk about four parts of the dashboard, as below:
+
+- [Status of Experiments](#status-of-experiments)
+- [Experiments](#experiments)
+  - [Create a New Experiment](#create-a-new-experiment)
+  - [Experiment detail](#experiment-detail)
+- [Events](#events)
+  - [Search Events](#search-events)
+- [Archive](#archive)
+- [At Last](#at-last)
+
+The pre-setup of this overview is:
+
+```bash
+minikube start # You can also use Kind
+
+kubectl create deployment hello-minikube --image=registry.cn-hangzhou.aliyuncs.com/google_containers/echoserver:1.10
+
+curl -sSL https://mirrors.chaos-mesh.org/latest/install.sh | bash
+
+kubectl port-forward -n chaos-testing svc/chaos-dashboard 2333:2333
+```
+
+Let's get started.
+
+## Status of Experiments
```

Comment:
For all headings in the body of this doc, please use the sentence-style capitalization (only capitalize the first word) to keep it consistent with existing documents. See https://chaos-mesh.org/docs/user_guides/run_chaos_experiment.

Reply:
Know about it. Already resolved.